server/src/modules/collections/collection-worker.service.ts

```diff
@@ -57,7 +57,8 @@ export class CollectionWorkerService extends TaskBase {
     await super.execute();
 
     // wait 5 seconds to make sure we're not executing together with the rule handler
-    setTimeout(() => {}, 5000);
```

Comment:
This isn't actually doing anything. If there's no reported issues I can remove my new wait below.

Reply:
I understand why it exists, it just wouldn't work like it was as setTimeout doesn't block. It runs the callback function after the timeout asynchronously